sdk/dns/Azure.ResourceManager.Dns/src/Generated/Models/DnsCnameRecordInfo.cs

```diff
@@ -11,7 +11,7 @@
 namespace Azure.ResourceManager.Dns.Models
 {
     /// <summary> A CNAME record. </summary>
-    internal partial class DnsCnameRecordInfo
+    public partial class DnsCnameRecordInfo
```

Comment:
why public?

Reply:
a DnsCnameRecordInfo formal parameter is in DnsRecordData's facotory method, so it needs to be public